srcpkgs/taskd/files/taskd/run

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 [ -r conf ] && . ./conf
-: ${TASKDDATA:=/var/lib/taskd}
-: ${TASKDUSER:=taskd}
+TASKDDATA="/var/lib/taskd"
+TASKDUSER=taskd
```

Comment:
This does something completely different, ": ${var:=foo}" sets the variable `var` to `foo` if its not set yet, allowing the `./conf` file to set variables or use the default values taskd and `/var/lib/taskd` if not defined in the `conf` file.

Reply:
Hmm, well the issue here is like the variable data is not being read during the command, as if the variable is null or so. At first I thought maybe it was possibly a permission issue, but turns out no. Setting it this way, the server did run. Could we, instead of using the TASKDDATA variable, and just use the /var/lib/taskd : 

taskd server --data /var/lib/taskd ....

Reply:
The variables work, this has to be something different. Other than this, I don't think the `user` keyword is supported by the `server` command and even if it would be, I think this is a different user than the user the service runs as.
`chpst` changes the uid/gid of the daemon, the `user` keyword is a user internal to taskd.